.github/actions/deploy-backend/action.yml

```diff
@@ -0,0 +1,29 @@
+name: Deploy Backend
+description: Deploy backend code to Cloud.gov environment.
+inputs:
+  USERNAME:
+    description: Cloud.gov user to authenticate with.
+    required: true
+  PASSWORD:
+    description: Cloud.gov authentication password.
+    required: true
+  ORG_NAME:
+    description: Cloud.gov Organization Name.
+    required: true # Default to our Org once setup.
+  SPACE_NAME:
+    description: Cloud.gov Space (environment) Name.
+    required: true
+  APP_NAME:
+    description: Applicaiton name (from manifest.yml).
+    required: true
+runs:
+  using: composite
+  steps:
+    - name: Deploy to Cloud.gov
+      run: |
+          export PATH=$HOME/bin:$PATH
+          cf login -a https://api.fr.cloud.gov -u ${{ inputs.USERNAME }} -p ${{ inputs.PASSWORD }} -o ${{ inputs.ORG_NAME }} -s ${{ inputs.SPACE_NAME }}
+          cf push ${{ inputs.APP_NAME }} -f manifest.yml
+          BACKEND_GUID=$(cf app opre-ops-test --guid)
```

Comment:
I'd like to try and further refactor the two deploy-* actions to a single one; but would need to work out a way to execute the extra `BACKEND_*` stuff; could do it conditionally, but feels ugly. For now, two actions still works. 

Will further enhance this with GH Environments as we need them.